.golangci.yml

```diff
@@ -4,13 +4,90 @@ run:
 #   timeout: 5m
 
 linters:
+  # Disable all linters.
+  # Default: false
   disable-all: true
+  # Enable specific linter
+  # https://golangci-lint.run/usage/linters/#enabled-by-default-linters
   enable:
+    - asciicheck
```

Comment:
oh wow, I was considering adding a grep command to CI for this. Glad it exists in golangci!